waku/README.md

```diff
@@ -0,0 +1,301 @@
+# Waku Whisper Specification
+
+> Version 0.0.5 (Draft)
+>
+> Authors: Oskar Thorén oskar@status.im, Dean Eigenmann dean@status.im
+
+## Table of Contents
+
+- [Abstract](#abstract)
+- [Motivation](#motivation)
+- [Specification](#specification)
+    - [Use of RLPx transport protocol](#use-of-rlpx-transport-protocol)
+    - [Packet Codes](#packet-codes)
+    - [Packet Format and Usage](#packet-format-and-usage)
+    - [Whisper Envelope](#whisper-envelope)
+    - [Contents of Data Field of the Message (Optional)](#contents-of-data-field-of-the-message-optional)
+    - [Payload Encryption](#payload-encryption)
+- [Light node](#light-node)
+    - [Rationale](#rationale)
+    - [Considerations](#considerations)
+- [Rationale](#rationale-1)
+- [Backwards Compatibility](#backwards-compatibility)
+    - [WakuWhisper bridging](#wakuwhisper-bridging)
+- [Forwards Compatibility](#forwards-compatibility)
+- [Security Considerations](#security-considerations)
+- [Implementation](#implementation)
+- [Changelog](#changelog)
+    - [Differences between shh/6 waku/0](#differences-between-shh6-waku0-1)
+- [Copyright](#copyright)
+
+## Abstract
+
+This specification describes the format of Waku messages within the ÐΞVp2p Wire Protocol. This spec substitutes [EIP- 627](https://eips.ethereum.org/EIPS/eip-627). Waku is a fork of the original Whisper protocol that enables better scalability and offline messaging, at the cost of some metadata protection guarantees. It does this through (a) light node support (b) mailserver (c) waku mode and (d) basic rate limiting.
+
+## Motivation
+
+<!-- This should be the motivation of Waku rather than the motivation of this spec: aka to enable scalability etc. -->
+
+It is necessary to specify the standard for Waku messages in order to ensure forward compatibility of different Waku clients, backwards compatibility with Whisper clients, as well as to allow multiple implementations of Waku.
```

Comment:
Backward compatibility will be through bridging, right? Should we clarify it here? Overall Waku is a new subprotocol so naturally it's not compatible with `shh`.

Reply:
@adambabik maybe we should take away the `backwards compatibility with Whisper clients` part, as the compatibility now seems semi-orthogonal due to the use of intermediate bridge nodes.

Reply:
The data format is compatible though, as evidenced by the fact that we can do bridging. How would people prefer to distinguish between these two forms of compatibility?